readinglist/resource.py

```diff
@@ -18,7 +18,7 @@ def wrapped_view(self, *args, **kwargs):
                 return view(self, *args, **kwargs)
             except RecordNotFoundError as e:
                 self.request.errors.add('path', 'id', six.text_type(e))
-                self.request.errors.status = "404 Resource Not Found"
+                self.request.errors.status = 404
```

Comment:
This is the normal Cornice behavior: http://cornice.readthedocs.org/en/latest/validation.html#changing-the-status-code-from-validators